warehouse/warehouse/resources/dataset_spec.yaml

```diff
@@ -61,15 +61,15 @@ tables:
         - clhcalipso
         - all
     day:
-        - pr_highfreq
+        - pr
```

Comment:
Removing _highfreq will only work with e2c code on master once Tom's PR is merged.

Reply:
I eliminated the "_highfreq" in the dataset_spec  many weeks ago (so that dataset_ids could be cleanly generated) and then adjusted for that in GenerateAtmDay, GenerateAtm3hr, with

>             vname = item['CMIP6 Name']
>             if vname in ['pr', 'rlut']:
>                 vname = f"{vname}_highfreq" # only for this day or 3hr job

So that e2c would be happy.  I will need to eliminate that once e2c takes care if it internally.

Reply:
P.S.  That "item['CMIP6 Name']" comes from "e3sm_to_cmip --info".